tests/Parallafka.Tests/ParallafkaBasicTests.cs

```diff
@@ -67,7 +78,7 @@ public async Task MaxMessageQueuedIsObeyed(int partitions, int maxMessagesQueued
             this._console.WriteLine($"Messages queued after stop: {consumer.MessagesQueued}");
             this._console.WriteLine($"Messages handled after stop: {messagesConsumed}");
 
-            Assert.True(messagesConsumed >= maxMessagesQueued && messagesConsumed <= maxMessagesQueued + 2);
+            Assert.True(consumer.MessagesQueued >= maxMessagesQueued && consumer.MessagesQueued <= maxMessagesQueued + 5);
```

Comment:
Comment on the +5?

Reply:
Running a few times, it seems to always be +2.  I don't know if that's a guarantee though.